### ISSUE TITLE
Jinja2Bear: Add support for indentation

### ISSUE THREAD

**Issue by ayan-b** [coala/coala-bears#2763]
_i.e._,
```
{% for a in b %}
  {{ a }}
{% endfor %}
``` 
is properly indented, whereas - 
```
{% for a in b %}
{{ a }}
{% endfor %}
``` 
is not.

**Comment by jayvdb** [comment]
The first one adds two space indentation.  That depends on the language of the generated file.

How should the language be determined, and the style of indentation choosen ?

**Comment by jayvdb** [comment]
(there are valid indentation features, but the originally stated problem is a nested language problem, and coala doesnt support that yet)

**Comment by ayan-b** [comment]
> The first one adds two space indentation. That depends on the language of the generated file.
> 
> How should the language be determined, and the style of indentation choosen ?

True. :sweat_smile:  We can start off by considering jinja syntax only _i.e_, indenting logical statements, for loops, set etc., but avoiding those which prints into the generated file.

**Comment by Naveenaidu** [comment]
@jayvdb  I would like to be assigned to this issue.

**Comment by Pihu1998** [comment]
Is this issue resolved?

**Comment by ayan-b** [comment]
@Pihu1998 there is an open PR #2803 .

**Comment by Naveenaidu** [comment]
> Is this issue resolved?

@Pihu1998  I am still working on this :see_no_evil:  - The core issue is fixed by the PR #2803 - Waiting for the review from the maintainers.

**Comment by Pihu1998** [comment]
@ayan-b  @Naveenaidu  Okay, thanks for the response!😃

<https://www.avast.com/en-in/recommend?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=default3&tag=e8d474e3-5ac4-4260-8207-0985f33f73e7>
I’m
protected online with Avast Free Antivirus. Get it here — it’s free forever.
<https://www.avast.com/en-in/recommend?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=default3&tag=e8d474e3-5ac4-4260-8207-0985f33f73e7>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
